### ISSUE TITLE
Q about finetune based on OmniVGGT's checkpoint

### ISSUE THREAD

**Issue by TwiceMao** [Livioni/OmniVGGT-official#28]
Thank you  @Livioni  for your excellent work. I'd like to fine-tune OmniVGGT's checkpoint while maintaining its Omni-Modality input characteristics. Should I modify https://github.com/Livioni/OmniVGGT-official/blob/5f02290543a1a0443652de828d6fddca64f6421f/configs/train.py#L19 to https://huggingface.co/Livioni/OmniVGGT/tree/main? However, this is online, and I'd like to use a locally loaded model. 
Also, my custom dataset has a depth mask; could you suggest which dataset would be most suitable for modification?

**Comment by Livioni** [comment]
Hi, thanks for your interest. 
Q1. Use "https://huggingface.co/Livioni/OmniVGGT/resolve/main/OmniVGGT.safetensors" shall be ok.
Q2. You can modify [dl3dv.py](https://github.com/Livioni/OmniVGGT-official/blob/op/omnivggt/datasets/dl3dv.py) to suit your needs. It has a similar mask operation. 
Thanks.
